src/python/example/diagram_vectorizations_distances_kernels.py

```diff
@@ -26,9 +26,9 @@
 
 LS = Landscape(resolution=1000)
 L = LS.fit_transform(diags)
-plt.plot(L[0][:1000])
-plt.plot(L[0][1000:2000])
-plt.plot(L[0][2000:3000])
+plt.plot(L[0][:999])
+plt.plot(L[0][999:2*999])
```

Comment:
why not `999:1999`?

Reply:
Actually, it should be `998` since the first **and** last coordinates are removed. Fixed it.

Reply:
Shouldn't your new `+ self.nan_in_range.sum()` mean that `resolution` always matches the number of bins, even after removing the extremes?

Reply:
You are totally right, I forgot to update the landscape resolution in some parts of the code. Hopefully it is good now.